common/djangoapps/edxmako/makoloader.py

```diff
@@ -2,7 +2,6 @@
 
 from django.conf import settings
 from django.template.base import TemplateDoesNotExist
-from django.template.loader import make_origin, get_template_from_string
```

Comment:
These functions have been moved to the Engine class. I haven't figured out the right fix yet but since they are used in only one specific case removing the imports allows the other logic in this file to work.